dockertest/environment.py

```diff
@@ -9,10 +9,7 @@
 
 import errno
 import subprocess
-# This always fails on travis with: no get in xattr (E0611)
-# dispite successful 'pip install pyxattr==0.5.1'.  A test
-# for this name has been added to .travis.yml directly.
-from xattr import get as getxattr  # pylint: disable=E0611
+import selinux
```

Comment:
You lost the `try`, that's why it broke. What I was suggesting was: keep the `try`, just ditch all the xattr crap in the exception handler